lib/activerecord-import/import.rb

```diff
@@ -529,6 +529,7 @@ def import_associations(models, options)
       # notes:
       #    does not handle associations that reference themselves
       #    should probably take a hash to associations to follow.
+      return if models.nil?
```

Comment:
Really the only time is if import is passed an array of columns and values, which is an invalid/unlikely scenario.

```
columns = [ :title, :author ]
values = [ ['Book1', 'FooManChu'], ['Book2', 'Bob Jones'] ]
Book.import columns, values, validate: false, recursive: true
```